templates/types/extractor/fastapi/app/ui/components/schema_editor.py

```diff
@@ -0,0 +1,28 @@
+import reflex as rx
+
+
+class SchemaState(rx.State):
+    schema: str
+
+    def handle_schema(self, schema: str):
+        self.schema = schema
+
+    def init_schema(self):
+        from app.api.models.output import Output
```

Comment:
ui layer must not import from api layer

Reply:
because this is a fullstack app in python so that why i moved the Output to a dedicated models folder. Do you mean that we shouldn't put it in the api folder?

Reply:
Yes exactly.
How about we have a service method with the parameters:
1. JSON schema as string 
2. Query
That we call from both API and UI.